fuzz/fuzz/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "fuzz"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+honggfuzz = "0.5.51"
+cairo-rs =  {path = "../"}
```

Comment:
The public api of cairo-rs has had some changes recently, I would advise pinning to a specific release rather than using relative path for the cairo-rs dependency

Reply:
done